backend/src/binaryInstaller.ts

```diff
@@ -40,76 +41,56 @@ class BinaryInstaller {
 
   async installBinary(installer: Installer) {
     return new Promise(async (resolve, reject) => {
-      var tmpDir = tmp.dirSync({ unsafeCleanup: true, keep: true })
-
-      // Stop and remove old binaries
-      await this.migrateBinaries(installer.binaryPath())
-
-      // Download and install binaries
-      await this.download(installer, tmpDir)
-
       const commands = new Command({ onError: reject, admin: true })
 
       if (environment.isWindows) {
         if (!existsSync(environment.binPath)) commands.push(`md "${environment.binPath}"`)
-        commands.push(`move /y "${installer.tempFile}" "${installer.binaryPath()}"`)
-        commands.push(`icacls "${installer.binaryPath()}" /T /C /Q /grant "*S-1-5-32-545:RX"`) // Grant all group "Users" read and execute permissions
+        commands.push(`icacls remoteit /T /C /Q /grant "*S-1-5-32-545:RX"`)
```

Comment:
Not sure you need this command... if you do it will probably need the .exe
```suggestion
        commands.push(`icacls remoteit.exe /T /C /Q /grant "*S-1-5-32-545:RX"`)
```